mwdb/app.py

```diff
@@ -165,6 +166,7 @@ def require_auth():
 api.add_resource(RefreshTokenResource, "/auth/refresh")
 api.add_resource(ValidateTokenResource, "/auth/validate")
 api.add_resource(AuthGroupListResource, "/auth/groups")
+api.add_resource(AuthFavoritesResource, '/auth/favorite')
```

Comment:
I would move this feature to `/object/<hash>/favorite` and implement the actions only for chosen object instead of operating on lists of objects.